### PR TITLE
[tempo-distributed] Tempo auth

### PR DESCRIPTION
**Pull request by sergeyshaykhullin**
Waiting for next tempo release

### CODE REVIEW

**Comment by sergeyshaykhullin** [comment]
@annanay25 @joe-elliott @mapno @mdisibio 1.0.0 released, this can be merged

**Review by mapno — APPROVED** [review]
LGTM, thanks!

File: charts/tempo-distributed/README.md [hunk 1]
```diff
@@ -250,13 +250,13 @@ Alternatively, in order to quickly test Tempo using the filestore, the [single b
...
-      ring:
-        kvstore:
-          store: memberlist
+  multitenancy_enabled: false
+  compactor:
+    compaction:
+      block_retention: 48h
+    ring:
+      kvstore:
+        store: memberlist
```
Comment (mapno) [inline charts/tempo-distributed/README.md:259]:
```suggestion
    multitenancy_enabled: false
    compactor:
      compaction:
        block_retention: 48h
      ring:
        kvstore:
          store: memberlist
```
Actually, the rest of the blocks have one more indentation. I think we need to add it to `multitenancy_enabled` or remove it from everywhere else as with the compactor block.

**Review by mapno — APPROVED** [review]
LGTM, thanks again!

**Pull request merged**